### DISCUSSION TITLE
Vercel's ip blocked by firehol

### DISCUSSION THREAD

**Discussion by gerhardsletten** [vercel/vercel#6182] [General]
Should really Vercel's IP 76.76.21.21 be blocked by firehol?

https://github.com/firehol/blocklist-ipsets/search?q=76.76.21.21

**Comment by fu-sen** [comment]
This information is likely to need to be raised.
This information seems to be worse than it was a few days ago.
Is Vercel being abused?

**Reply by gerhardsletten** [reply]
Sent them a support-issue on this now

**Comment by fu-sen** [comment]
Really bad situations are spreading. Added by Graylist of Project Turris. 
https://project.turris.cz/en/greylist

This may increase the number of people who have problems with Vercel references...

**Reply by fu-sen** [reply]
This list has been deleted.

**Comment by fu-sen** [comment]
The coinbl list hasn't been updated for 2 weeks, so we have to wait.
https://gitlab.com/ZeroDot1/CoinBlockerLists

**Reply by fu-sen** [reply]
Some have been updated on May 30th, but unfortunately they are still listed.

**Comment by fu-sen** [comment]
You can see a little more specific refusal information here. 
https://www.abuseipdb.com/check/76.76.21.21

There have been no reports since the beginning of June, but it seems that it is still affected by being on the list in May.

**Comment by cronin4392** [comment]
I am running into an issue with this as well. A client's ISP has the Vercel IP blacklisted. This is a huge issue for us.

**Comment by EduardoBautista** [comment]
@gerhardsletten @cronin4392 @fu-sen 

I submitted this PR:

https://github.com/firehol/blocklist-ipsets/pull/197

Please show support for the PR to get it merged ASAP. If you find any other repos with this IP address, I highly recommend sending a PR removing it.

**Reply by mwt** [reply]
good job!

**Reply by EduardoBautista** [reply]
Actually, looks like it that PR didn't do anything. Apparently those files are autogenerated. They take a list of domain names from:

https://gitlab.com/ZeroDot1/CoinBlockerLists/-/tree/master

Firehol then resolves the domain names to IP addresses and creates those IP address lists. This is just an issue with firehol which takes a bad approach to security. Unless you can convince organizations to stop using firehol, I am afraid there is nothing we can do.

**Reply by mwt** [reply]
oh, well it doesn't seem to be on the list anymore. So, something must have happened:
https://api.malware-research.org/v1/coinblockerlist/76.76.21.21

**Comment by offero** [comment]
Major business partners of ours are now starting to block this IP address. It's not a good situation and is impacting our ability to conduct business.

**Reply by mwt** [reply]
Do you know where they're getting it from? It's been weeks since it was removed from the lists.

**Reply by fu-sen** [reply]
It is still left in this search result. The removal is part and probably we have forgotten to deal with these.
https://github.com/firehol/blocklist-ipsets/search?q=76.76.21.21

**Comment by offero** [comment]
I will ask. Can you provide any documentation that might exist so that I
can demonstrate that the issue is resolved?

**Comment by Wandergin** [comment]
We are also facing this issue. Our hosting provider has blocked vercel IP and it is on their blacklist, what is being done to resolve this issue? We will be forced to move off Vercel and as it is a crucial issue. It's likely caused by people using vercel for malicious purposes, as shown in the link:

 https://www.abuseipdb.com/check/76.76.21.21?page=1#report
